lib/html.js

```diff
@@ -474,14 +474,14 @@ var addAttributes = function(elem, attr) {
 var appendDOM = function(elem, child) {
   if (child) {
     var tag = (elem.tagName||'').toLowerCase();
-    if (elem.nodeType === 8) { // comment
-      if (child.nodeType === 3) { // text node
+    if (elem.nodeType === window.Node.COMMENT_NODE) {
+      if (child.nodeType === window.Node.TEXT_NODE) {
```

Comment:
I was on the fence about referencing `Node` as an implicit global

Reply:
Yeah, I think this is better